### ISSUE TITLE
Do Javascript libraries need to be included in the standard_header.php?

### ISSUE THREAD

**Issue by gregcorbett** [GOCDB/gocdb#256]
As identified while reviewing #140 

javascript libraries seemingly get included by both the standard header and by individual pages.

i.e. `jquery.multiple.select.js` is included on `htdocs/web_portal/static_php/standard_header.php` and by `htdocs/web_portal/views/site/view_all.php`.

This duplication feels unnecessary, and the standard header is supposedly static php, but needs further investigation.

**Comment by GRyall** [comment]
I agree there is a need to standardise this and the duplication is unnecessary. Whether this should be on pages or in the header feels like a question of direction of travel of GOCDB - the more pages that make use of the libraries the more it makes sense to standardise and include them in the header.

**Comment by gregcorbett** [comment]
Looking into this a little bit with @RoseECooper, it seems ordering of the includes is important.

i.e. `moment-with-locales.min.js` needs to be included before `bootstrap-datetimepicker.min.js` is included, else the datetime picker functionality doesn't work. This order feels like a pain to unpick for each page where JS is used.

So maybe we move the includes from individual pages to the standard header.

**Comment by gregcorbett** [comment]
From speaking someonie who knows more about JS than me, I don't believe there is a best practice reason to move the includes out of the standard header and having them in the standard header seems more maintainable.
